src/arch/linux/mapping/mapping.c

```diff
@@ -433,7 +462,10 @@ void *mmap_mapping(int cap, void *target, size_t mapsize, int protect)
       (cap & (MAPPING_DPMI|MAPPING_VGAEMU|MAPPING_INIT_LOWRAM|MAPPING_KVM)))
     flags = _MAP_32BIT;
 #endif
-  addr = mmap(target, mapsize, protect,
+  if ((cap & (MAPPING_INIT_LOWRAM|MAPPING_KVM)) && flags != MAP_FIXED)
```

Comment:
`!= MAP_FIXED`?
Maybe `(!(flags & MAP_FIXED))`?

Reply:
sure. This is just because of `int flags = (target != (void *)-1) ? MAP_FIXED : 0;` higher up. Will change.

Reply:
I also forgot to pass _MAP_32BIT! Which caused the CI errors. Fixed now.